### PR TITLE
LPS-177253 | Test Fix | Master

### PR DESCRIPTION
**Pull request by JhossephG**
[LPS-177253](https://issues.liferay.com/browse/LPS-177253)

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
Note: I just need run the SF in one of these commits, but at friday it wasn't working

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/macros/Picklist.macro [hunk 1]
```diff
@@ -120,8 +120,6 @@ definition {
 	}
 
 	macro assertDisabledPicklistName {
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/macros/Picklist.macro:122]: In the future, we should delete this macro entirely. It's one step and only an assertion so it should be done on the testcase level.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPermission.testcase [hunk 2]
```diff
@@ -1,4 +1,6 @@
 @component-name = "portal-object"
+definition {
+@component-name = "portal-object"
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPermission.testcase:3]: This is what's causing your SF failure. You have duplicated lines (1-2 and 3-4) and therefore an unclosed boundary "{}". Please fix.

**Review by epark18 — COMMENTED** [review]
@JhossephG Thanks for this! Please review my comment and fix your SF when you have a chance. Also, really great job attaching all of those index files/test results. Very helpful

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPermission.testcase (hunk 2 above)
Comment (JhossephG) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectPermission.testcase:3]: Oh man, thanks for this. I really don't noticed this 😅

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**